cache.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+import requests
+import json
+from json import encoder
+import sys
+
+import six
+from tqdm import tqdm
+
+import ec2
+
+
+def add_pretty_names(instances):
+    family_names = {
+        "t4g": "T4g General Purpose Graviton",
+        "t3": "T3 General Purpose",
+        "t2": "T2 General Purpose",
+        "t1": "T1 Previous generation: (not recommended)",
+        "m6g": "M6g  General Purpose Graviton",
+        "m5": "M5 General Purpose",
+        "m4": "M4 General Purpose",
+        "m3": "M3 Previous generation: (not recommended)",
+        "m2": "M2 General Purpose",
+        "m1": "M1 Previous generation: (not recommended)",
+        "r6gd": "R6gd Memory optimized (w/ data tiering)",
```

Comment:
for elasticache r6gd vs r6g, would it be helpful to include that r6gd has disk storage as the differentiator? I know that would make the name super long.

Reply:
We can do that, yes. We're thinking about an "instance family" feature where each instance has a link that explains what all specs it has but this is a good placeholder.